src/System/Linux/IO/URing/Sqe.hsc

```diff
@@ -155,6 +156,17 @@ pollAdd fd events userd = do
     setUserData userd
     pokeField #{poke struct io_uring_sqe, poll_events} events
 
+-- | Cancel a previously issued poll request.
+pollRemove
+  :: UserData
+  -> UserData
+  -> SqeBuilder ()
+pollRemove toCancel userd = do
+    zeroIt
+    setOpCode (#const IORING_OP_POLL_REMOVE)
+    setAddr $ plusPtr nullPtr (fromIntegral toCancel)
```

Comment:
A (slightly) less hacky way to accomplish this would be to rather use
```haskell
     setAddr $ wordPtrToPtr $ fromIntegral toCancel
```

Reply:
:+1: